packages/fxa-content-server/app/scripts/views/pair/pairing-totp-mixin.js

```diff
@@ -6,7 +6,7 @@ import AuthErrors from '../../lib/auth-errors';
 
 export default function () {
   return {
-    checkTotpStatus() {
+    checkTotpStatus(sessionToken) {
```

Comment:
The passing in of the sessionToken but depending on the account from `this.getSignedInAccount()` seems strange to me. How would the sessionToken returned by getSignedInAccount be different to the one passed in?

Is it possible for the browser to return an account with no sessionToken which does not cause the signedInAccount to update it's sessionToken?